pkg/process/v6/transformers/entry.go

```diff
@@ -9,21 +9,27 @@ import (
 )
 
 type RelatedEntries struct {
-	VulnerabilityHandle grypeDB.VulnerabilityHandle
+	VulnerabilityHandle *grypeDB.VulnerabilityHandle
+	Provider            *grypeDB.Provider
 	Related             []any
 }
 
 func NewEntries(models ...any) []data.Entry {
 	var entry RelatedEntries
 
-	for _, model := range models {
+	for i := range models {
+		model := models[i]
 		switch m := model.(type) {
 		case grypeDB.VulnerabilityHandle:
-			entry.VulnerabilityHandle = m
+			entry.VulnerabilityHandle = &m
 		case grypeDB.AffectedPackageHandle:
 			entry.Related = append(entry.Related, m)
 		case grypeDB.AffectedCPEHandle:
 			entry.Related = append(entry.Related, m)
+		case grypeDB.KnownExploitedVulnerabilityHandle:
+			entry.Related = append(entry.Related, m)
+		case grypeDB.Provider:
```

Comment:
I think adding `Provider` isn't part of this PR, but was just missed before? Or am I missing something?

Reply:
it is part of this PR -- related to https://github.com/anchore/grype-db/pull/507#discussion_r1965648901